addons/mail/static/src/models/message_composition/message_composition.js

```diff
@@ -0,0 +1,212 @@
+/** @odoo-module **/
+
+import { emojis } from '@mail/js/emojis';
+import { registerModel } from '@mail/model/model_core';
+import { unlink, replace } from '@mail/model/model_field_command';
+import { attr, many2many, one2one } from '@mail/model/model_field';
+import { addLink, escapeAndCompactTextContent, parseAndTransform } from '@mail/js/utils';
+
+registerModel({
+    name: 'MessageComposition',
```

Comment:
We didn't take the habit to put docstring above new model register.
Could be worth start doing it. Same motivation as with some fields that are one-line comments: `camelCase` naming are not as easy to get as it seems, so a plain English text helps quite a lot to understand things quickly.

For instance, just telling that this model represent a message being composed helps already.

Reply:
Would be nice a small comment that briefly explain what this model is about.
Something like:

```
Model of a message being composed in a composer.

Note that a message composition is not a full-fledged message,
because it hasn't been posted yet.
Still, it looks somewhat like a message, which is intended because
when a message composition is posted, it becomes a message.
```

1st paragraph just in plain english what's this model, and 2nd paragraph to clarify nuance between message composition and message.